docs/source/pages/tutorials/use_verification_summary_attestation.rst

```diff
@@ -0,0 +1,159 @@
+                }
+            }
+            ],
+            "predicateType": "https://slsa.dev/verification_summary/v1",
+            "predicate": {
+            "verifier": {
+                "id": "https://github.com/oracle/macaron",
+                "version": {
+                "macaron": "0.10.0"
+                }
+            },
+            "timeVerified": "2024-08-25T06:36:24.654718+00:00",
+            "resourceUri": "pkg:maven/io.micronaut/micronaut-core@4.6.3-oracle-00001",
+            "policy": {
+                "content": "#include \"prelude.dl\"\n\nPolicy(\"gdk_provenance_policy\", component_id, \"Policy for GDK builds\") :-\n    check_passed(component_id, \"mcn_provenance_expectation_1\").\n\napply_policy_to(\"gdk_provenance_policy\", component_id) :-\n    is_component(component_id, purl),\n    match(\"^pkg:maven/io.micronaut/micronaut-core@.*$\", purl)."
+            },
+            "verificationResult": "PASSED",
+            "verifiedLevels": []
+            }
+        }
+
+
+
+The VSA adheres to the `schema <https://slsa.dev/spec/v1.0/verification_summary>`_ provided by SLSA. However, rather than specifying a URI for the policy, it includes the policy directly within the VSA under the ``predicate.policy.content`` field. Below is a pretty-printed format of the policy as it appears in the VSA.
+
+.. toggle::
+
+    .. code-block:: prolog
+
+        #include "prelude.dl"
+
+        Policy("gdk_provenance_policy", component_id, "Policy for GDK builds") :-
+            check_passed(component_id, "mcn_provenance_expectation_1")
+
+        apply_policy_to("has-hosted-build", component_id) :-
+            is_component(component_id, purl),
+            match("^pkg:maven/io.micronaut/micronaut-core@.*$", purl).
+
+This policy makes sure the :ref:`mcn_provenance_expectation_1 <checks>` check, which verifies the content of the provenance file matches :ref:`CUE expectation <pages/using:Verifying provenance expectations in CUE language>`. You can find the template policy files for GDK builds below:
+
+* `Template CUE expectation <https://github.com/oracle/macaron/tree/main/src/macaron/resources/policies/gdk/expectation.cue.template>`_
+* `Template Datalog policy file <https://github.com/oracle/macaron/tree/main/src/macaron/resources/policies/gdk/policy.dl.template>`_
+
+The VSA also includes the list of subjects and their corresponding checksums that have been verified, the version of Macaron used, the timestamp of the verification, and the result of the verification.
+
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+Automatically check the artifact checksum and verification result
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To verify that the artifact checksum matches the subject listed in the VSA and that the verification process has passed, follow these steps:
+
+
+**Download the check_vsa.sh script:**
```

Comment:
Do you think we should let the reader now the requirements to run this script properly? For example:
- The Bash version
- Tools like `shasum` and `awk` available in `PATH`

Reply:
I have checked and these are the standard utilities that are by default available. But I can add them. There is no specific requirement for the Bash version.

Reply:
See commit [32e104c](https://github.com/oracle/macaron/pull/858/commits/32e104c12ea49dec6675910dc6e9a3ea59baf4d9)